src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -144,9 +147,15 @@
     <Compile Include="System\Data\SqlTypes\SQLString.cs" />
     <Compile Include="System\Data\SqlTypes\SQLUtility.cs" />
     <Compile Include="System\Data\SqlTypes\SqlXml.cs" />
-	<Compile Include="$(CommonPath)\System\NotImplemented.cs"/>
+    <Compile Include="$(CommonPath)\System\NotImplemented.cs" />
+    <Compile Include="Interop\LocaleInterop.cs" />
+    <Compile Include="Interop\ILocaleApiHelper.cs" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
+    <Compile Include="Interop\LocaleWindowsApiHelper.cs" />
```

Comment:
Originally I thought the same way. But since this Helper does different thing, it seems it will be clearer to name this as what it does. One is using WindowsAPI interop. and the other is using hardcoded mapper. If in the future, windows also take this API, we can use just 1 helper. or if there is other way in unix which using file / registry look up, we can simply add / remove different implementation. 

We also did this similar case in SNI. e.g we have SNINative and SNI manage code. 
Instead of call this SNI.Unix or SNI.Windows.

.Unix or .Windows will be used if we have 2 same implementation but different data. e.g EncodingTable.Windows.cs and EncodingTable.Unix.cs.
Both of them are encoding table, and the only different is one is for unix and the other is for windows. So in that case, it will be make sense to use that convention .Unix /.Windows.